Dockerfile

```diff
@@ -6,11 +6,14 @@ ARG GITHUB_TOKEN
 RUN apt update && apt install -y git procps
 RUN pip install --upgrade pip
 
+RUN pip install jupyter
+
+RUN echo 'alias la="ls -la"' >> ~/.bashrc
+RUN echo 'alias jn="jupyter notebook --ip 0.0.0.0 --allow-root --no-browser --port=8889"' >> ~/.bashrc
+
 # install python package
 WORKDIR /openclsim
 ADD . /openclsim
 RUN pip install -e .
 
-EXPOSE 8888
```

Comment:
This is the default port for jupyter. Why are you removing this/changing it to 8889?

Reply:
at van oord reserve 8888 for notebooks running on the host (anaconda), and while we use 8889 for developing with vscode in containers so you can run host and container notebooks simultaneously.